### PR TITLE
Fix type annotation in babel-code-frame

### PR DESCRIPTION
**Pull request by forivall**
Also add a one-sentence description to the readme

### CODE REVIEW

File: packages/babel-code-frame/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # babel-code-frame
+
+Pretty prints a segment of code, primarily for use in error messages.
```
Comment (sebmck) [inline packages/babel-code-frame/README.md]: This should be in a separate commit.

Comment (forivall) [inline packages/babel-code-frame/README.md]: @sebmck fixed (amended). see below.

**Comment by sebmck** [comment]
Actually just realised this is targeting the `development` branch. This is my local development copy for v6 and it's under extreme flux and not everything is being regularly committed. I've already refactored this module and fixed the type annotation as the compiler codebase is being migrated to being flow type checked. I'd recommend not submitting any PRs targeting this branch as nobody is really supposed to be touching it but me. Thank you though :blush:

**Comment by forivall** [comment]
:point_right: :hammer:
